graphics/macgui/macbutton.cpp

```diff
@@ -123,13 +181,18 @@ bool MacButton::draw(bool forceRedraw) {
 		break;
 	}
 	case kRound:
-		Graphics::drawRoundRect(r, 4, 0, _active, _wm->getDrawPixel(), &pd);
+		Graphics::drawRoundRect(r, 4, 0, false, _wm->getDrawPixel(), &pd);
```

Comment:
That was because i'm using the invertOuter to indicate the active state. If we are active now, then we will invert it by using invertOuter. so for the RoundButton, we just draw the line, and if it's active, then we will invert it. looks like we can use _active to replace _invertOuter

Reply:
ok, i've changed that part. Use _active instead of _invertOuter to represent state and control inverting outer